cinch/roles/jenkins_slave/templates/swarm.upstart.conf

```diff
@@ -0,0 +1,21 @@
+description "Jenkins slave swarm client"
```

Comment:
I could go either way, I just got in the habit of leaving off the .j2 because it doesn't seem helpful that I can tell. It's already possible to discern that the files are templates based on the directory structure.